nni/runtime/protocol.py

```diff
@@ -32,8 +32,7 @@ class CommandType(Enum):
         _in_file = open(3, 'rb')
         _out_file = open(4, 'wb')
 except OSError:
-    _msg = 'IPC pipeline not exists, maybe you are importing tuner/assessor from trial code?'
-    logging.getLogger(__name__).warning(_msg)
+    pass
```

Comment:
why remove warning info?

Reply:
It causes more problem than benefit. Importing the package in wrong place won't cause real problem.